app/routes/teams.rb

```diff
@@ -31,6 +31,9 @@ class Teams < Core
       # Create a new team.
       post '/teams/new' do
         please_login
+        temp_usernames = params[:team]["usernames"].split(",")
+        temp_usernames.delete(current_user.username)
+        params[:team]["usernames"] = temp_usernames.join(",")
         team = Team.by(current_user).defined_with(params[:team], current_user)
```

Comment:
Also here, if you pushed the logic to handle the current user inviting themselves to their own team into[ `Team#invite`](https://github.com/exercism/exercism.io/blob/3de39152058110ace5b1ea98f3f9915d13480294/lib/exercism/team.rb#L73), that logic will be reused in [`Team#defined_with`](https://github.com/exercism/exercism.io/blob/3de39152058110ace5b1ea98f3f9915d13480294/lib/exercism/team.rb#L50) so you'd get the security of not inviting the team creator on both team creation and new user adds.